R/exercise.R

```diff
@@ -304,10 +304,13 @@ evaluate_exercise <- function(
 
   # Copy files from data directory into exercise
   if (is.null(data_dir)) {
-    # First check environment variable, then default to "data/" if it exists
-    source_dir <- Sys.getenv(
-      "TUTORIAL_DATA_DIR",
-      unset = if (dir.exists("data")) "data" else ""
+    # First check options(), then environment variable, then default to "data/"
+    source_dir <- getOption(
+      "tutorial.data_dir",
+      default = Sys.getenv(
+        "TUTORIAL_DATA_DIR",
+        unset = if (dir.exists("data")) {"data"} else {""}
+      )
```

Comment:
The braces snuck back in and it's okay not to name the second argument of `getOption()` and `Sys.getenv()`

```suggestion
    source_dir <- getOption(
      "tutorial.data_dir",
      Sys.getenv("TUTORIAL_DATA_DIR", if (dir.exists("data")) "data" else "")
```